src/core/internal/convert.d

```diff
@@ -37,7 +37,7 @@ const(ubyte)[] toUbyte(T)(const ref T val) if (__traits(isFloating, T) && (is(T
 {
     if (__ctfe)
     {
-        static if (T.mant_dig == float.mant_dig || T.mant_dig == double.mant_dig)
+        static if (floatFormat!T == FloatFormat.Float || floatFormat!T == FloatFormat.Double)
```

Comment:
```suggestion
        static if (T.sizeof == float.sizeof || T.sizeof == double.sizeof)
```

Reply:
Does this really matter given that floatFormat is likely instantiated anyway, so we're just going to hit the template cache?

Reply:
Even if it matters it probably doesn't matter.